migrations/1685459848689.do.remove-timezones.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE retrieval_results
+  ALTER COLUMN start_at TYPE TIMESTAMP WITHOUT TIME ZONE,
+  ALTER COLUMN first_byte_at TYPE TIMESTAMP WITHOUT TIME ZONE,
+  ALTER COLUMN end_at TYPE TIMESTAMP WITHOUT TIME ZONE;
```

Comment:
Can we merge these two migrations into a single one, please, so that we have only one migration per PR?